server/influxdb.go

```diff
@@ -0,0 +1,43 @@
+package server
+
+import (
+	"context"
+
+	influxdb2 "github.com/influxdata/influxdb-client-go/v2"
+)
+
+const (
+	org                      = "trento"
```

Comment:
Most probably these values shouldn't be constants, at least the `org` and `bucket`.
I would make them come from cmd or env variables.

Anyway, easy to change in the future!